resources/prosody-plugins/mod_fmuc.lua

```diff
@@ -458,3 +459,22 @@ local function iq_from_main_handler(event)
     return true;
 end
 module:hook('iq/host', iq_from_main_handler, 10);
+
+
+function filter_stanza(stanza)
```

Comment:
Can you add a note on what this filters? I'm trying to wrap my head around it, but I can't. The filter is attached to the s2s sessions? Filters stanzas being sent? Does returning `nil` result in the stanza being dropped?